softdevice_controller/include/sdc_hci_vs.h

```diff
@@ -1050,6 +1067,27 @@ uint8_t sdc_hci_cmd_vs_set_adv_randomness(const sdc_hci_cmd_vs_set_adv_randomnes
  */
 uint8_t sdc_hci_cmd_vs_compat_mode_window_offset_set(const sdc_hci_cmd_vs_compat_mode_window_offset_set_t * p_params);
 
+/** @brief Set APR handling witin LE Power Control Request.
+ *
+ * This command enables APR handling within scope of the LE Power Control Request
+ * procedure by the Link Layer.
+ *
+ * When APR handling is enabled, the controller applies received APR to the local
+ * power settings.
+ * See Core_v.5.4, Vol 6, Part D, Section 6.27, Fig 6.57.
+ * and Core_v.5.4, Vol 6, Part B, Section 5.1.17.1
```

Comment:
```suggestion
 * See Core_v.5.4, Vol 6, Part D, Section 6.27, Fig 6.57,
 * and Core_v.5.4, Vol 6, Part B, Section 5.1.17.1.
```
Just a minor fix to be in line with these updates: https://github.com/nrfconnect/sdk-nrfxlib-next/pull/84/

Reply:
I've introduced the fixes, you can take a look now.